client/pages/profilePage.jsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import Navbar from '../components/navbar';
+import Photostream from '../components/photostream';
+
+export default class ProfilePage extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      userId: this.props.userId,
```

Comment:
Avoid sticking props in your state. Sometimes you need to, but those are special situations. If you can just use `this.props` elsewhere instead of `this.state`, use the props.